packages/material-ui/src/Tab/Tab.js

```diff
@@ -81,6 +81,7 @@ export const styles = theme => ({
   /* Styles applied to the label container element if `label` is provided. */
   labelContainer: {
     width: '100%', // Fix an IE 11 issue
+    textAlign: 'center',
```

Comment:
Should we move the style to the `root` style rule?
As far as I understand the issue, we are depending
on the default button style (`text-align: center`)

Reply:
Good point.